### DISCUSSION TITLE
Pass multiple arguments to global fetcher

### DISCUSSION THREAD

**Discussion by IdoBouskila** [vercel/swr#2676] [Q&A] [answered]
Hello 👋🏽
Is it possible to pass multiple arguments to global fetcher.
for example:
``` js
<SWRConfig value={ {
 fetcher: () => fetchEntities(entitiesId, endpoint)
} }>
```

Thanks!

**Comment by StyleShit** [comment, marked as answer]
When you need to pass multiple arguments to the fetcher, you need to make sure that the key also gets changed based on those arguments or the cache key will be wrong and you'll get invalid/stale data.

You can simply construct the cache key as an object with all of the dependencies of the fetcher:

https://swr.vercel.app/docs/arguments.en-US#multiple-arguments
